blog-post.html

```diff
@@ -1,12 +1,159 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Blog Post</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-  </body>
-</html>
+
+<head>
+  <title>BLOG ZONE</title>
+  <link href='./style.css' rel='stylesheet' />
+  <meta charset="utf-8" />
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Aladin&family=Rock+Salt&display=swap');
+    @import url('https://fonts.googleapis.com/css2?family=BenchNine:wght@300&family=Pathway+Gothic+One&display=swap');
+  </style>
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+
+
+
+<body>
+  <!--- start header!-->
+   <!--- start main grid-->
+
+  <div class="blogpostcontainer">
+   
+    <div class="sidebar-blank1"></div>
+     <a id="blog-zone" href="./index.html">
+      <div class="header">
+        <p><br><br>

```

Comment:
Using the `<br>` tags here are one way of going about doing this, Another thing you could have done is to add `margin-top` to your class of header to push the text down. Using `margin-top` would be a more common way of going about it